### PR TITLE
Add documentation about billing

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

File: billing.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+   Official prices are available at https://symfony.com/cloud/pricing.
+
+When and how is my project billed?
+----------------------------------
+
+You project is billed monthly based on the subscription creation date (when the
+project has been created) or based on the trial end.
+
+Billing is done prepaid: when the invoice is issued you are  paying for the
```
Comment (fabpot) [inline billing.rst:14]: Billing occurs at the beginning of each period

Comment (fabpot) [inline billing.rst:14]: double spaces

File: billing.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+upcoming period. For example, if your project is created on July 2nd without
+trial, an invoice is issued for the selected plan for the period from July 2nd
+to August 2nd.
+
+If you change you plan or addons durring the billing period, `prorations`_ are
```
Comment (fabpot) [inline billing.rst:19]: during

File: billing.rst [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+going to be applied to your next invoice. In the eventuality your project has
```
Comment (fabpot) [inline billing.rst:20]: If your project

File: billing.rst [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+important pending items, an intermediary invoice can be issued at any time at
+our sole discretion.
```
Comment (fabpot) [inline billing.rst:22]: issued before the beginning of the next period.

File: billing.rst [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+.. _prorations:
+
+How are managed prorations?
+---------------------------
+
+SymfonyCloud relies on Stripe to compute prorations down the second. Stripe will
```
Comment (fabpot) [inline billing.rst:29]: down to

File: billing.rst [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+compute the remaining time until the end of the current billing period, add a
```
Comment (fabpot) [inline billing.rst:30]: Stripe compute

File: billing.rst [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+negative item to the invoice for the previous monthly amount prorated to the
+reamining time and add another item for the new monthly amount also prorated.
```
Comment (fabpot) [inline billing.rst:32]: remaining

File: billing.rst [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+The following invoice will include the billing for the upcoming billing period,
```
Comment (fabpot) [inline billing.rst:33]: The next invoice?

File: billing.rst [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+plus the proration lines for the previous upcoming billing period.
```
Comment (fabpot) [inline billing.rst:34]: previous billing period?

File: billing.rst [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+
+.. note::
+
+   Switching plan back and forth will generate several of these proration lines
```
Comment (fabpot) [inline billing.rst:38]: missing end dot.

File: billing.rst [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+
+If you do not upscale your project or accept billing, you project will be
+automatically suspended at the end of your trial. If you do no pay in the days
+following its suspension, your project will be automatically deleted.
+
+What is the difference between development and production plans?
+----------------------------------------------------------------
+
+The development plan is intended to let you test and prepare your project before
+it going live. You do not have dedicated resources and can not attach production
```
Comment (fabpot) [inline billing.rst:73]: before going live

File: billing.rst [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+domains to your Main environment.
```
Comment (fabpot) [inline billing.rst:74]: "main" environment?

File: billing.rst [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+
+How do I change my project plan?
+--------------------------------
+
+To upscale your project one step up (from Standard to Medium for example), use
+``symfony project:scale``. To downscale one step down, use ``symfony
+project:scale --down``.
+
+You can also scale directly to a specific plan using ``symfony project:scale
+<plan>`` (replace ``plan`` with the plan name).
```
Comment (fabpot) [inline billing.rst:88]: Link to the list of plans? Or list them here?

File: billing.rst [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+
+How do I add more storage to my project?
+----------------------------------------
+
+Your subscription comes with 5GB of persistent storage per environment. Adding
+more storage is made per increment of 5GB and is billed $3.60 / 3,00€ per
+increment and per environment.
+
+Add more storage using ``symfony project:edit storage X`` (replacing ``X`` by
+the desired storage amount).
```
Comment (fabpot) [inline billing.rst:98]: units?

File: billing.rst [hunk 15]
```diff
@@ -0,0 +1,146 @@
...
+
+How do I add more environments to my project?
+---------------------------------------------
+
+Your subscription comes with 3 non-production and active environments. You can
```
Comment (fabpot) [inline billing.rst:103]: Is it 3 non-prod envs or 1 prod env + 2 non-prod? In any case, we should mention the prod env.

File: billing.rst [hunk 16]
```diff
@@ -0,0 +1,146 @@
...
+add more environment by increment of 3, billed $25 / 21€ per increment.
```
Comment (fabpot) [inline billing.rst:104]: environments

File: billing.rst [hunk 17]
```diff
@@ -0,0 +1,146 @@
...
+
+Add more environments using ``symfony project:edit environments X`` (replacing
+``X`` by the desired environments quota).
```
Comment (fabpot) [inline billing.rst:107]: by the number of environments

File: billing.rst [hunk 18]
```diff
@@ -0,0 +1,146 @@
...
+
+You can cancel your SymfonyCloud subscription at any time using ``symfony
+project:delete``. You will be prompted if you want to delete your project
+immediately or automatically at the end of your current billing period.
+
+How can I change my billing details?
+------------------------------------
+
+At the moment it is not possible to edit your billing details yourself. If you
+need to edit them, please file a ticket support at
```
Comment (fabpot) [inline billing.rst:120]: change?

File: billing.rst [hunk 19]
```diff
@@ -0,0 +1,146 @@
...
+https://symfony.com/cloud/support.
```
Comment (fabpot) [inline billing.rst:121]: You should add the query string to auto-select SymfonyCloud in the form

File: billing.rst [hunk 20]
```diff
@@ -0,0 +1,146 @@
...
+
+How can I change a project owner?
+---------------------------------
+
+Changing a project owner is possible but requires manual intervention from our
+staff, please contact us at https://symfony.com/cloud/support.
```
Comment (fabpot) [inline billing.rst:127]: same here for the query string

File: billing.rst (hunk 19 above)
Comment (tucksaun) [reply billing.rst:121]: `/cloud/support` (vs `/support`) already does that

File: billing.rst [hunk 21]
```diff
@@ -0,0 +1,147 @@
...
+
+Your project is billed monthly based on the subscription creation date (when the
+project has been created) or based on the trial end.
+
+Billing occurs at the beginning of each period, when the invoice is issued you
+are paying for the upcoming period. For example, if your project is created on
+July 2nd without trial, an invoice is issued for the selected plan for the
+period from July 2nd to August 2nd.
+
+If you change you plan or addons during the billing period, `prorations`_ are
```
Comment (fabpot) [inline billing.rst:19]: you**r** plan

File: billing.rst [hunk 22]
```diff
@@ -0,0 +1,147 @@
+Billing FAQ
+===========
+
+.. note::
+
+   Official prices are available at https://symfony.com/cloud/pricing.
```
Comment (fabpot) [inline billing.rst:6]: Can we make it a link?

File: billing.rst [hunk 23]
```diff
@@ -0,0 +1,147 @@
...
+
+How do I change my project plan?
+--------------------------------
+
+To upscale your project one step up (from Standard to Medium for example), use
+``symfony project:scale``. To downscale one step down, use ``symfony
+project:scale --down``.
+
+You can also scale directly to a specific plan using ``symfony project:scale
+<plan>`` (replace ``plan`` with the plan name from https://symfony.com/cloud/pricing).
```
Comment (fabpot) [inline billing.rst:88]: Can we have a proper link?

File: billing.rst [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+
+How do I add more storage to my project?
+----------------------------------------
+
+Your subscription comes with 5GB of persistent storage per environment. Adding
+more storage is made per increment of 5GB and is billed $3.60 / 3,00€ per
```
Comment (fabpot) [inline billing.rst:94]: I would remove all prices in this document and link to the pricing page instead.

File: billing.rst [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+
+Add more storage using ``symfony project:edit storage X`` (replacing ``X`` by
+the desired storage amount in Gigabytes).
+
+How do I add more environments to my project?
+---------------------------------------------
+
+Your subscription comes with 1 main and 3 non-production and active
+environments. You can add more non-production environments by increment of 3,
+billed $25 / 21€ per increment.
```
Comment (fabpot) [inline billing.rst:105]: remove billed ...

File: billing.rst (hunk 19 above)
Comment (fabpot) [reply billing.rst:122]: Should be a proper link

File: billing.rst (hunk 20 above)
Comment (fabpot) [reply billing.rst:128]: here as well, proper link

**Pull request merged**